### PR TITLE
Fix 4 issues

### PR DESCRIPTION
**Pull request by Jeff-Stammler**
Blinking is now "Erica" style
Log X/Y are now WHOLE numbers
Log Time is now 1 decimal point.
Log StartTime in "d" file to help with de-duping.

also cleaned up Match layout (centered buttons vertically, padding, etc.)
got rid of "Time:" and just show the countdown clock.  saves room.

### CODE REVIEW

File: app/src/main/java/com/cpr3663/cpr_scouting_app/Logger.java [hunk 1]
```diff
@@ -217,16 +219,14 @@ public void LogEvent(int in_EventId, float in_X, float in_Y, boolean in_NewSeque
         // If this is NOT a new sequence, we need to write out the previous event id that goes with this one
         if (!in_NewSequence) prev = String.valueOf(seq_number_prev);
         
-        // Determine string values for x, y and time. Round them to 1 decimal places.
+        // Determine string values for x, y and time. Truncate them.
         // If they happen to be whole numbers, trim off the ".0"
```
Comment (AdrielStammler) [inline app/src/main/java/com/cpr3663/cpr_scouting_app/Logger.java:223]: Shouldn't you get rid of this comment.

File: app/src/main/java/com/cpr3663/cpr_scouting_app/data/Events.java [hunk 2]
```diff
@@ -31,9 +31,9 @@ public class Events {
 
     // Constructor
     public Events() {
-        event_list = new ArrayList<EventRow>();
-        auto_events = new ArrayList<String>();
-        teleop_events = new ArrayList<String>();
+        event_list = new ArrayList<>();
+        auto_events = new ArrayList<>();
+        teleop_events = new ArrayList<>();
```
Comment (AdrielStammler) [inline app/src/main/java/com/cpr3663/cpr_scouting_app/data/Events.java:36]: Why? Is it just irrelevant to tell it what it an ArrayList of and if that is the reason why not leave it in to make sure its the right type?

**Review by AdrielStammler — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/cpr3663/cpr_scouting_app/Logger.java (hunk 1 above)
Comment (Jeff-Stammler) [reply app/src/main/java/com/cpr3663/cpr_scouting_app/Logger.java:223]: yes i should.

File: app/src/main/java/com/cpr3663/cpr_scouting_app/data/Events.java (hunk 2 above)
Comment (Jeff-Stammler) [reply app/src/main/java/com/cpr3663/cpr_scouting_app/data/Events.java:36]: Android Studio "complains" that it's unnecessary.  It knows the type, so when you call "new" you don't need to supply it.  I think this is a common Java practice.

File: app/src/main/java/com/cpr3663/cpr_scouting_app/Logger.java (hunk 1 above)
Comment (Jeff-Stammler) [reply app/src/main/java/com/cpr3663/cpr_scouting_app/Logger.java]: Fixed.

File: app/src/main/java/com/cpr3663/cpr_scouting_app/data/Events.java (hunk 2 above)
Comment (AdrielStammler) [reply app/src/main/java/com/cpr3663/cpr_scouting_app/data/Events.java:36]: Oh, ya, I kind of forgot you already told it the type when you defined it.

**Review by AdrielStammler — APPROVED** [review]

**Pull request merged**